tests/list/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -82,10 +82,8 @@ list(
     "veryVeryVeryVeryVeryLongKey" => $veryVeryVeryVeryVeryLongValue
 ) = $data[0];
 list(
-    "veryVeryVeryVeryVeryVeryVeryVeryVeryLongKey" =>
-        $veryVeryVeryVeryVeryVeryVeryVeryVeryLongValue,
-    "veryVeryVeryVeryVeryVeryVeryVeryVeryLongKey" =>
-        $veryVeryVeryVeryVeryVeryVeryVeryVeryLongValue
+    "veryVeryVeryVeryVeryVeryVeryVeryVeryLongKey" => $veryVeryVeryVeryVeryVeryVeryVeryVeryLongValue,
```

Comment:
unsure why these changed exactly... (it changed even before i simplified the printing logic) - guessing there was an update in how `list` was parsed or something?

Reply:
Seems as though its the same as other keyed array logic now though

Reply:
@mgrip looks like due https://github.com/glayzzle/php-parser/issues/150, feel free to fix in this PR

Reply:
Ah good to know, thanks! What do you guys think the "correct" formatting should be? I tried adding the line break to our "entry" node printing, but it changed a lot of the other tests

Reply:
@mgrip i think linebreak is part of other issue. We should add newlines as in PSR-12, but now we only add after namespace. Better create new issue around mewlines as in PSR-12 (example issue around newline for php tag https://github.com/prettier/plugin-php/issues/367)

Reply:
👍 sounds good. so good to merge this as is then?

Reply:
Let's fix this change around `list` in this PR, because it is regression and we can merge. Also don't forget create issue :smile:

Reply:
Sorry what are we fixing exactly though? To change this back, we'd have to change the formatting around the "entry" type, which is going to affect a lot of other tests

Reply:
@mgrip means we should do this here, PR should not have regressions after release, even in master

Reply:
@evilebottnawi should be good now